pandas/_libs/interval.pyx

```diff
@@ -423,7 +423,15 @@ cdef class Interval(IntervalMixin):
             msg = '`other` must be an Interval, got {other}'
             raise TypeError(msg.format(other=type(other).__name__))
 
+        # TODO:: use Interval.is_empty() if/when implemented
+        def is_empty(intv):
```

Comment:
needs a doc-string / typing. I think we should just make this a proper method, with proper tests in this PR.